lua/starfall/libs_sh/entities.lua

```diff
@@ -300,50 +300,54 @@ function ents_methods:getLinkedComponents()
 	return list
 end
 
---- Parents or unparents an entity. Only holograms can be parented to players and, in clientside, only clientside holograms can be parented.
+--- Parents or unparents an entity. Only holograms can be parented to players and clientside holograms can only be parented in the CLIENT realm.
 -- @param Entity? parent Entity parent (nil to unparent)
--- @param number|string? attachment Optional attachment name or ID
+-- @param number|string? attachment Optional attachment name or ID.
 -- @param number|string? bone Optional bone name or ID. Can't be used at the same time as attachment
 function ents_methods:setParent(parent, attachment, bone)
 	local child = getent(self)
-	if CLIENT and debug.getmetatable(child)~=SF.Cl_Hologram_Meta then SF.Throw("Can only setParent clientside holograms in the clientside!", 2) end
 	checkpermission(instance, child, "entities.setParent")
-	if attachment~=nil and bone~=nil then SF.Throw("Can't have both attachment and bone args set!", 2) end
+	if CLIENT and debug.getmetatable(child) ~= SF.Cl_Hologram_Meta then SF.Throw("Only clientside holograms can be parented in the CLIENT realm!", 2) end
+	if attachment ~= nil and bone ~= nil then SF.Throw("Arguments `attachment` and `bone` are mutually exclusive!", 2) end
+	
 	if parent ~= nil then
 		parent = getent(parent)
-		if parent:IsPlayer() and not child.IsSFHologram then SF.Throw("Can only setParent holograms onto players!", 2) end
-		local param, type
-		if bone~=nil then
+		if parent:IsPlayer() and not child.IsSFHologram then SF.Throw("Only holograms can be parented to players!", 2) end
+		
+		local type, param
+		if attachment ~= nil then
```

Comment:
Pls revert the reordering of this if statement so I can see what actually changed